### ISSUE TITLE
Lerna run isn't running on packages after upgrading to v.3.14.x and above

### ISSUE THREAD

**Issue by alcferreira** [lerna/lerna#2165]
## Expected Behavior

It seems that `lerna run` isn't running on packages from a monorepo.
Until v3.13.4 it used to work fine

## Current Behavior

It is not working after v3.14.x

## Steps to Reproduce (for bugs)

1. Clone this [repo](https://github.com/CraveFood/farmblocks)
2. Install dependencies `yarn`
3. Run [`yarn build`](https://github.com/CraveFood/farmblocks/blob/master/package.json#L11) > which will create a [lib directory](https://github.com/CraveFood/farmblocks/blob/master/packages/alert/package.json#L21) on every package 
4. Check out if the lib directory has been created for the alert package: `/farmblocks/packages/alert/lib`
5. Remove this directory
6. Upgrade lerna to the latest version (or v3.14.x) `yarn add --dev lerna@latest -W` 
7. Run `yarn build` again. The directory `/farmblocks/packages/alert/lib` should've been created but it was not.

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
{
  "lerna": "3.13.1",
  "packages": ["packages/*"],
  "version": "independent",
  "command": {
    "publish": {
      "conventionalCommits": true,
      "message": "chore(release): [ci skip] publish"
    }
  },
  "npmClient": "yarn",
  "useWorkspaces": true
}
```
</p></details>

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.15.0 |
| `yarn --version` | 1.16.0 |
| `node --version` | 12.1.0 |

| OS | Version |
| --- | --- |
| macOS Mojave | 10.14 |

**Comment by danez** [comment]
Probably the same issue as #2107

**Comment by evocateur** [comment]
@alcferreira [v3.16.0](https://github.com/lerna/lerna/releases/tag/v3.16.0) contains the fix in #2107, can you try it and verify if this is fixed? Thanks!

**Comment by alcferreira** [comment]
@evocateur and @danez fix confirmed, thank you all!
